src/pybind/mgr/mirroring/module.py

```diff
@@ -0,0 +1,92 @@
+from typing import List
+
+from mgr_module import MgrModule, Option
+
+from .fs.snapshot_mirror import FSSnapshotMirror
+
+class Module(MgrModule):
+    COMMANDS = [
+        {
+            'cmd':  'fs snapshot mirror enable '
+                    'name=fs,type=CephString ',
+            'desc': 'enable snapshot mirroring for a filesystem',
+            'perm': 'rw'
+        },
```

Comment:
i'd suggest use `CLICommand` decorator for defining commands. see https://docs.ceph.com/en/latest/mgr/modules/#exposing-commands and https://github.com/ceph/ceph/blob/0e7a7be5c065bd7f792b378a59c14b503370e394/src/pybind/mgr/devicehealth/module.py#L163-L170

Reply:
ack